test/client/index.js

```diff
@@ -467,7 +469,7 @@ module.exports = config => {
             headers,
             options
           )
-            .then(function (response) {
+            .then(response => {
               // Update to the new verified tokens
               this.sessionToken = response.sessionToken
               this.keyFetchToken = response.keyFetchToken
```

Comment:
This callback was unbound so we were actually creating two new global variables called `sessionToken` and `keyFetchToken` here. Introducing strict mode changed that behaviour to throw and I've fixed the problem using fat arrow.